content/blog/meetup-kit.md

```diff
@@ -0,0 +1,11 @@
+---
+title: The gRPC Meetup Kit
+date: "2017-09-14"
```

Comment:
General nit: we don't need quotes around dates.

Reply:
I'll go through and reformat the rest of the blog posts. A lot of unnecessary things in there!

Reply:
Yeah, thanks for doing that! 🎉